pt_miniscreen/tile_group_context.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+
+from .event import AppEvents, post_event
+from .tile_group import TileGroup
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class TileGroupContext:
+    tile_groups: List[TileGroup]
+    name: str = ""
+    index: int = 0
+
+    @property
+    def current_tile_group(self):
+        return self.tile_groups[self.index]
+
+    def go_to_next_tile_group(self):
+        logger.debug("Going to next tile group")
+
+        self.current_tile_group.active = False
+        self.index = (self.index + 1) % len(self.tile_groups)
+        self.current_tile_group.active = True
+
+    def can_go_to_next_tile_group(self):
+        return len(self.tile_groups) > 1
```

Comment:
Why not just have this as a return case in the 'go to next' method and use directly? One less thing to call